packages/lib/src/components/Giftcard/Giftcard.tsx

```diff
@@ -48,7 +48,7 @@ export class GiftcardElement extends UIElement {
     }
 
     get displayName() {
-        return this.props.name;
+        return this.props.brandsConfiguration[this.props.brand]?.name || this.props.name;
```

Comment:
Don't we need to add the typescript definition for this new property/configuration?

Reply:
👍